flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartition.java

```diff
@@ -52,6 +55,10 @@
 	/** Flag indicating whether the subpartition has been released. */
 	private volatile boolean isReleased;
 
+	/** The number of non-event buffers currently in this subpartition */
```

Comment:
nit: Should also end with a period.

Reply:
alright - created a fixup for FLINK-7468 here